### ISSUE TITLE
can you write complete tutorial?

### ISSUE THREAD

**Issue by dev0zero** [jinmingjian/sourcekite#11]
where i need get this framework ??

/usr/lib/sourcekitdInProc.framework

why when i download this project and try build a project it searching SourcektdInPorc in /usr/lib/rd ??

im install this swift instead official
https://github.com/norio-nomura/docker-sourcekit-builder/releases

nothing happens

**Comment by andradei** [comment]
Agreed. The instructions for installation are vague, probably due the fact that the author doesn't own a Macbook. I couldn't manage to get it working either because I don't know what to do with the link provided for `sourcekitdInProc.framework`.

It points to a docker project with swift installed and that thing included. But how do we use it to get sourcekite working?
